src/TikaOnDotNet.Tests/text_extraction.cs

```diff
@@ -55,141 +102,211 @@ public void should_be_able_to_delete_the_mp4_after_extraction()
+            var data = File.ReadAllBytes("files/Tika.xls");
+            var textExtractionResult = _textExtractor.Extract(data);
 
             textExtractionResult.Text.Should().Contain("Use the force duke");
         }
 
         [Test]
-        public void should_extract_from_doc()
+        public void should_extract_from_xlsx()
         {
-            var textExtractionResult = _cut.Extract("files/Tika.doc");
+            var textExtractionResult = _textExtractor.Extract("files/Tika.xlsx");
 
-            textExtractionResult.Text.Should().Contain("formatted in interesting ways");
+            textExtractionResult.Text.Should().Contain("Use the force duke");
         }
 
         [Test]
-        public void should_extract_from_ppt()
+        public void should_extract_mp4()
         {
-            var textExtractionResult = _cut.Extract("files/Tika.ppt");
+            var textExtractionResult = _textExtractor.Extract("files/badgers.mp4");
 
-            textExtractionResult.Text.Should().Contain("This document is used for testing");
+            textExtractionResult.ContentType.Should().Be("video/mp4");
         }
 
         [Test]
-        public void should_extract_from_xls()
+        public void should_extract_msg()
         {
-            var textExtractionResult = _cut.Extract("files/Tika.xls");
+            var textExtractionResult = _textExtractor.Extract("files/test.msg");
+            
+            textExtractionResult.Text.Should().Contain("This is my test file");
+        }
 
-            textExtractionResult.Text.Should().Contain("Use the force duke");
+        #region OCR test
+        [Test]
+        public void should_ocr_tif_with_tesseract()
+        {
+            if (!File.Exists(Path.Combine(TesseractPath, "tesseract.exe")))
+                Assert.Ignore("Tesseract not found in the path '" + TesseractPath + "'");
+            else
+            {
+                var textExtractionResult = _textExtractor.Extract("files/testfile.tif");
+                textExtractionResult.Text.Should().Contain("This is some example text that should be ocred by tesseract");
+            }
         }
 
         [Test]
-        public void should_extract_from_xls_with_byte()
+        public void should_ocr_jpg_with_tesseract()
         {
-            var data = File.ReadAllBytes("files/Tika.xls");
-            var textExtractionResult = _cut.Extract(data);
+            if (!File.Exists(Path.Combine(TesseractPath, "tesseract.exe")))
+                Assert.Ignore("Tesseract not found in the path '" + TesseractPath + "'");
+            else
+            {
+                var textExtractionResult = _textExtractor.Extract("files/testfile.png");
+                textExtractionResult.Text.Should().Contain("This is some example text that should be ocred by tesseract");
+            }
+        }
 
-            textExtractionResult.Text.Should().Contain("Use the force duke");
+        [Test]

```

Comment:
Please move the tesseract tests to their own fixture. There is a different class under test.